### PR TITLE
Additional VSCode and Poetry tasks added

### PR DESCRIPTION
**Pull request by esfoobar-usds**

### CODE REVIEW

File: data/data-pipeline/.vscode/launch.json [hunk 1]
```diff
@@ -9,63 +9,100 @@
...
       "type": "python",
       "request": "launch",
       "module": "data_pipeline.application",
-      "args": ["score-full-run"]
+      "args": [
+        "score-full-run"
+      ]
+    },
+    {
+      "name": "Score Geo",
```
Comment (switzersc-usds) [inline data/data-pipeline/.vscode/launch.json:53]: what does this do exactly? wondering if there's a verb we can use that'll make this clearer

**Review by switzersc-usds — CHANGES_REQUESTED** [review]
one small change request to use a more descriptive or longer name for "Score Geo". right now I'd have to look at the docs or code to figure out what it does, so I think the name should give more of a hint

File: data/data-pipeline/.vscode/launch.json (hunk 1 above)
Comment (esfoobar-usds) [reply data/data-pipeline/.vscode/launch.json:53]: That is what we called the "Combine" lambda function. Should we call it "Combine Score and GeoJSON"?

**Review by switzersc-usds — APPROVED** [review]
🎉 thanks!

**Pull request merged**